keras_hub/src/utils/keras_utils.py

```diff
@@ -104,3 +104,40 @@ def running_on_gpu():
         import torch
 
         return torch.cuda.is_available()
+
+
+def gpu_supports_fused_attention_op():
+    deny_list = ["T4"]
+    available_gpu = [gpu.upper() for gpu in get_gpu_names()]
+    for denied_gpu in deny_list:
+        if any(denied_gpu in gpu for gpu in available_gpu):
+            return False
+    return True
+
+
+def get_gpu_names():
```

Comment:
Maybe drop a docstring or comment explaining what types of strings are actually returned on different frameworks? Sounds like this is pretty not normalized, so good to flag that gotcha for future users on the function.